modules/services/wob.nix

```
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) getExe mkEnableOption mkIf mkOption mkPackageOption optional;
+
+  cfg = config.services.wob;
+  settingsFormat = pkgs.formats.ini { };
+
+  configFile = settingsFormat.generate "wob.ini" cfg.settings;
+in {
+  meta.maintainers = with lib.maintainers; [ Scrumplex ];
+
+  options.services.wob = {
+    enable = mkEnableOption "wob";
+    package = mkPackageOption pkgs "wob" { };
+
+    settings = mkOption {
+      type = settingsFormat.type;
+      default = { };
+      example = {
+        "" = {
+          border_size = 10;
+          height = 50;
+        };
+        "output.foo".name = "DP-1";
+        "style.muted".background_color = "032cfc";
+      };
```

I think this needs to be a literal expression since it otherwise will be shown as a single line in the documentation:

```suggestion
      example = literalExpression ''
        {
          "" = {
            border_size = 10;
            height = 50;
          };
          "output.foo".name = "DP-1";
          "style.muted".background_color = "032cfc";
        }
      '';
```